### DISCUSSION TITLE
Add option for (physical) security key for unlocking the app

### DISCUSSION THREAD

**Discussion by FeliksMarksen** [ente-io/ente#4500] [Enhancements]
I love that you can setup a passkey with a security key for 2FA for the account. It would be great if there also was the option to unlock the app only with the security key.

**Comment by tastyspacedragon** [comment]
Adding the link to this thread so people can upvote on the same thread

Passwordless login using PRF Passkeys - Click here then upvote: https://github.com/ente-io/ente/discussions/10494
